app/src/test/java/com/jerboa/UtilsKtTest.kt

```diff
@@ -0,0 +1,137 @@
+package com.jerboa
+
+import androidx.compose.ui.unit.dp
+import com.jerboa.datatypes.api.GetUnreadCountResponse
+import org.junit.Assert.*
+import org.junit.Test
+
+class UtilsKtTest {
+    @Test
+    fun testCalculateCommentOffset() {
+        assertEquals(0.dp, calculateCommentOffset(0, 0))
+        // assertEquals()
```

Comment:
Remove

Reply:
Oops, good catch. I was going to try to hit the other branch, then forgot to come back.